dev/MenuFlyout/MenuFlyout_19h1_themeresources.xaml

```diff
@@ -257,7 +257,8 @@
                             Foreground="{ThemeResource MenuFlyoutItemKeyboardAcceleratorTextForeground}"
                             HorizontalAlignment="Right"
                             VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-                            Visibility="Collapsed" />
+                            Visibility="Collapsed"
+                            AutomationProperties.AccessibilityView="Raw" />
```

Comment:
Please make change to all files from menuFlyout_rs1_themeresources to menuFlyout_rs5_themeresources